OBAKit/UI/OBATheme.m

```diff
@@ -52,6 +54,43 @@ + (void)setAppearanceProxies {
     [[[UITableViewCell appearance] contentView] setPreservesSuperviewLayoutMargins:YES];
 }
 
++ (UIView *)createNavbarTitleViewWithTitle:(NSString *)title subtitle:(NSString *)subtitle andStyle:(OBAAppearanceNavBarTitleViewStyle)style {
```

Comment:
@aaronbrethorst The fonts defined in `OBATheme` are too big on the navigation bar.